src/Reference.php

```diff
@@ -118,7 +128,7 @@ public function getModel($defaults = []) : Model
         // if model is Closure, then call it and return model
         if (is_object($this->model) && $this->model instanceof \Closure) {
             $c = $this->model;
-            $c = $c($this->owner, $this, $defaults);
+            $c = ($c)($this->owner, $this, $defaults);
```

Comment:
what this change does?

Reply:
I think it executes closure with arguments. Interesting syntax

Reply:
probably same thing, i don't mind either way.

Reply:
sorry the idea was to remove all "yellow" and "red" syntax notice.
is only a sort of much correct syntax.

```php
$c = ($this->model)($this->owner, $this, $defaults);
return $this->addToPersistence($c, $defaults);
```
no changes, at functional level just syntax.